include/sensors/core/tasks/pressure_driver.hpp

```diff
@@ -173,10 +173,8 @@ class MMR920C04 {
     }
 
     auto send_pressure(uint32_t message_index) -> void {
-        auto pressure =
-            mmr920C04::Pressure::to_pressure(_registers.pressure.reading);
         auto pressure_fixed_point =
-            convert_to_fixed_point(pressure, S15Q16_RADIX);
+            convert_to_fixed_point(pressure_pascals, S15Q16_RADIX);
```

Comment:
can we put some comments in the send pressure functions that we are sending over pascals? We should probably also document [this in confluence](https://opentrons.atlassian.net/wiki/spaces/RPDO/pages/3513712720/Pressure+Sensor).